work_queue/src/bindings/python3/work_queue.binding.py

```diff
@@ -2073,12 +2075,33 @@ class RemoteTask(Task):
     # @param coprocess  The name of the coprocess which has the function you wish to execute. The coprocess should have a name() method that returns this
     # @param
     # @param command    The shell command line to be exected by the task.
-    # @param kwargs	    keyword arguments used in function to be executed by task. An optional kwarg exec_method can be specified to one of fork, thread, and direct to choose between those three methods of execution
-    def __init__(self, fn, coprocess, **kwargs):
+    # @param args       positional arguments used in function to be executed by task. Can be mixed with kwargs
+    # @param kwargs	    keyword arguments used in function to be executed by task. 
+    def __init__(self, fn, coprocess, *args, **kwargs):
         Task.__init__(self, fn)
-        event = json.dumps(kwargs)
-        Task.specify_buffer(self, event, "infile")
+        self._event = {}
+        self._event["work_queue_kwargs"] = kwargs
+        self._event["work_queue_positional_args"] = args
         Task.specify_coprocess(self, coprocess)
+    ##
+    # Specify function arguments as a dictionary argument. This function overrides and kwargs and args passed to the task previously
+    # @param self             Reference to the current remote task object
+    # @param *args            Positonal args to passed to the function 
+    # @param argument_dict    A dictionary that contains arguments by keyword. The key is the name of the argument and the value is the value that should be passed to the argument
+    def specify_argument_dictionary(self, *args, argument_dict=None):
```

Comment:
`specify_fn_args(args=None, kwargs=None)` as it does more than the dict. Use like: `specify_fn_args([1,2,3], {"a": 2})`